lib/OrdinaryDiffEqCore/src/solve.jl

```diff
@@ -71,6 +71,7 @@ function DiffEqBase.__init(
         initialize_integrator = true,
         alias_u0 = false,
         alias_du0 = false,
+        alias = ODEAliases(),
```

Comment:
```suggestion
        alias = ODEAliases(alias_u0 = alias_u0, alias_du0=false),
```

It needs the deprecation path.